examples/pytorch_lightning/barlowtwins.py

```diff
@@ -55,7 +55,7 @@ def configure_optimizers(self):
     num_workers=8,
 )
 
-gpus = 1 if torch.cuda.is_available() else 0
+accelerator = "gpu" if torch.cuda.is_available() else "cpu"
 
-trainer = pl.Trainer(max_epochs=10, gpus=gpus)
+trainer = pl.Trainer(max_epochs=10, devices=1, accelerator=accelerator)
```

Comment:
Why not `devices="auto"`, `accelerator="auto"` ? This also applies at other places. Or just remove the arguments completely.

Reply:
> Why not devices="auto", accelerator="auto" ? This also applies at other places. Or just remove the arguments completely.

`"auto"` is not supported for some older PyTorch Lightning versions 😞